index.css

```diff
@@ -87,3 +87,8 @@ button {
   box-shadow: var(--box-shadow-lg);
   transition: all 0.3s ease;
 }
+@media screen and (max-width: 640px) {
+  .form-section.form-scale {
+    display: none;
+  }
```

Comment:
Look at the solution on Code Monster to see how we can make the logo appear to jump to the top of the login form on smaller screens.